### PR TITLE
Weave save and load

### PR DESCRIPTION
**Pull request by lilleswing**
Changes to make WeaveTensorGraph save/loadable

BatchNormalization didn't implement save_tensors/none_tensors.

WeaveLayer was attempting to save activation and initialization functions not strings.

### CODE REVIEW

File: deepchem/models/tensorgraph/graph_layers.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def __init__(self,
 
     """
     super(WeaveLayer, self).__init__(**kwargs)
-    self.init = initializations.get(init)  # Set weight initialization
-    self.activation = activations.get(activation)  # Get activations
+    self.init = init  # Set weight initialization
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/graph_layers.py]: save strings

File: deepchem/models/tensorgraph/graph_layers.py [hunk 2]
```diff
@@ -183,7 +185,7 @@ def none_tensors(self):
     self.W_AA, self.b_AA, self.W_PA, self.b_PA, self.W_A, self.b_A = None, None, None, None, None, None
 
     out_tensor, out_tensors, trainable_weights, variables = self.out_tensor, self.out_tensors, self.trainable_weights, self.variables
-    self.out_tensor, self.out_tensors, self.trainable_weights, self.variables, self.activation, self.init = None, [], [], [], None, None
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/graph_layers.py]: remove none-ing activation and init as they are no longer functions

File: deepchem/models/tensorgraph/layers.py [hunk 3]
```diff
@@ -2929,6 +2929,14 @@ def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
       self.out_tensor = out_tensor
     return out_tensor
 
+  def none_tensors(self):
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/layers.py]: Needed to implement for BatchNormalization

**Comment by rbharath** [comment]
@lilleswing Great fix!

LGTM

**Pull request merged**